en/upgrade-tidb-operator.md

```diff
@@ -40,6 +40,23 @@ This document describes how to upgrade TiDB Operator and Kubernetes.
     helm upgrade tidb-operator pingcap/tidb-operator --version=${version} -f ${HOME}/tidb-operator/${version}/values-tidb-operator.yaml
     ```
 
+    After all of pods start normally, execute:
```

Comment:
```suggestion
    After all the Pods start normally, execute the following command to update TiDB Operator:

    {{< copyable "shell-regular" >}}
```

Reply:
`After all the Pods start normally, execute the following command to check the image of TiDB Operator:`